src/components/MyThoughts.js

```diff
@@ -0,0 +1,212 @@
+import Lightbulb from './images/Lightbulb.jpg'
+import Drumroll from './images/Drumroll.jpg'
+
+/* Component that that prints 2 articels side by side, and will adjust to one above one below if its a small screen */
+
+const MyThoughts = ({ color, colorSecond }) => {
+  return (
+    <OuterWrapper>
+      <Wrapper>
+        <SectionHeader color={color}>MY THOUGHTS</SectionHeader>
+        <FeaturedProjectWrapper>
+          <ProjectCard
+            href="https://medium.com/@viktor.tommy.svensson/early-midlife-crises-couldnt-afford-ferrari-so-changed-carriers-instead-fd1b03b75c32"
+            target="_blank">
+            <ThumbnailWrapper url={Lightbulb} />
+            <ProjectSmallHeader>OKT 2022</ProjectSmallHeader>
+            <ClonedProjectInfoHeader color={colorSecond}>
+            Early midlife crises, couldn’t afford a Ferrari so changed carriers instead
+            </ClonedProjectInfoHeader>
+            <ClonedProjectInfo>{`Well I’m going to be honest here, it’s not really a midlife crisis, more like a midlife “awakening”.
+            I realised that I was not comfortable at my job, not happy. And that’s a dangerous state for the mind, and for the endless possiblites that’s out there.`}<SubParagraphArrows>{' >>>'}</SubParagraphArrows>
+            </ClonedProjectInfo>
+          </ProjectCard>
+          <ProjectCard
+            href="https://medium.com/@viktor.tommy.svensson/early-midlife-crises-couldnt-afford-ferrari-so-changed-carriers-instead-fd1b03b75c32"
+            target="_blank">
+            <ThumbnailWrapper url={Drumroll} />
+            <ProjectSmallHeader>OKT 2022</ProjectSmallHeader>
+            <ClonedProjectInfoHeader color={colorSecond}>
+            Changing paths: how to stop living your life on autopilot
+            </ClonedProjectInfoHeader>
+            <ClonedProjectInfo> {`What's autopilot to you? is it driving home and barely knowing how you got there? Or are you like me who knows there current job like the back of your hand, and every "one liner" you deliver
+                                kills you a bit inside(yes i worked in retail)`}<SubParagraphArrows>{' >>>'}</SubParagraphArrows>
+            </ClonedProjectInfo>
+          </ProjectCard>
+        </FeaturedProjectWrapper>
+      </Wrapper>
+    </OuterWrapper>
+  );
+};
+
+export default MyThoughts;
+
+/* Components local styled components */
+
+const ClonedProjectInfoHeader = styled(ProjectInfoHeader)`
+  margin-block-start: 0;
+  margin-block-end: 0.7%;
+`;
+
+const ClonedProjectInfo = styled(ProjectInfo)`
+  margin-block-start: 0;
+  margin-block-end: 0;
+  color: #183a1d;
+`;
+
+const ProjectSmallHeader = styled.p`
+  font-size: 24px;
+  line-height: 1.4;
+  color: #f0a04b;
+  margin-block-start: 0.5%;
+  margin-block-end: 0.5%;
+
+  @media (max-width: 800px) {
+  font-size: 16px;
+  line-height: 1.4;
+}
+
+`;
+
+/* const Wrapper = styled.div`
+  width: 80%;
```

Comment:
Has this part maybe been moved to the Global Styling component? And can be removed here?